decidim-meetings/decidim-meetings.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |s|
   s.add_dependency "kaminari", "~> 1.0.1"
 
   s.add_development_dependency "decidim-dev", Decidim.version
+  s.add_development_dependency "decidim-proposals", Decidim.version
```

Comment:
Because this way you could have the meetings engine without the proposals one (this why I added the ResourceLink concept). But it's needed at dev in order to test it.